src/Umbraco.Web/Models/ContentEditing/DocumentTypeCollectionDisplay.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Umbraco.Web.Models.ContentEditing
+{
+    public class DocumentTypeCollectionDisplay
```

Comment:
Although there are several other models suffixed with the term `Display`, we need to be aware of what that suffix is referring to. Not all models should be suffixed with this such as `DocumentTypeCollectionDisplay`. A `Display` model is one that is used for displaying an item/content for editing to the user. We also have `Save` models which are used to POST the values back to the server. Simple models such as this shouldn't have the `Display` suffix and should just be named with something that makes sense. 

This model also needs to have the appropriate Json.Net attributes applied just like our other models so that the results are not returned in `TitleCase` and instead in `camelCase`. I'll change this in v8 since we can't change it now in v7 since it will be a breaking change.

Reply:
Any reason we don't just let json.net serialize camel case automatically https://www.newtonsoft.com/json/help/html/NamingStrategyCamelCase.htm?

Reply:
We can do that if we assign a custom serializer to our base class webapi controllers (we can't apply the normal pattern in docs because that applies this globally).  We started out with attributes because we didn't want to interfere with global settings since a long time ago we didn't know how to do it for specific controllers and we've just continued down that path. We could apply the custom serialization to our base classes but it just never became a priority and would need some testing to make sure that our custom attributes our still used with this setting since we can't have the setting mucking around with casing that already exists. (i.e. in this example if we apply that serializer, we'll need to change the JS since the casing will change)